### ISSUE TITLE
Animated stickers bugged in the Telegram Desktop on Linux 

### ISSUE THREAD

**Issue by OverwhelmingFire** [telegramdesktop/tdesktop#6308]
### Steps to reproduce
1. Open any chat. Notice that animated stickers are displayed incorrectly.
2. Open any animated sticker pack by pressing on a sticker OR press this button:
![Screenshot from 2019-07-16 15-15-00](https://user-images.githubusercontent.com/34778977/61307200-6ecc3b00-a7dd-11e9-8ec1-849b85234c4f.png)

### Expected behaviour
A box containing the animated stickerpack shows up in the bottom part of the Telegram client's window. OR, if the smile button was pressed, the window expands to the right and shows the box containing a list of user's stickerpacks. The client doesn't crash and the stickers are displayed correctly.

### Actual behaviour
The client silently crashes if there are any animated stickers in the opened box. No crash logs available. 

### Configuration
**Operating system:** Debian GNU/Linux 9 (stretch)

**Version of Telegram Desktop:** 1.7.14

**Used theme**: classic

<details><summary><b>How are animated stickers displayed
</b>:

![Screenshot from 2019-07-16 16-28-53](https://user-images.githubusercontent.com/34778977/61313618-53b3f800-a7ea-11e9-92bd-64ae341110f5.png)

</summary>
</details>

**Comment by msva** [comment]
Did you install "official" (blob) telegram client, or the one from deibain repos?

**Comment by OverwhelmingFire** [comment]
> Did you install "official" (blob) telegram client, or the one from debian repos?

The official one, since I've just downloaded the archived client&updater from the official site and unpacked it. And launched.

**Comment by john-preston** [comment]
@OverwhelmingFire Can you try latest beta version? Switch “Install beta versions” in settings and download the update. Does it work fine?

**Comment by mikkovedru** [comment]
In case someone encounters the same bug, I will tell how I fixed it.

- Computer: Mac.
- Operating System: Ubuntu.

The core problem was that Telegram was installed from the Ubuntu repository (via install software functionality). And that version didn't even have the option to "Install beta versions” in Settings/Advanced like it supposed to.

So what I did was I removed that version of Telegram and installed the one from the official site.

One caveat: 64-bit Telegram won't work on Mac and therefore a 32-bit version of the Telegram Desktop needs to be downloaded.

Animated stickers started working in the downloaded version. In addition, the “Install beta versions” options appeared in the Settings in case I will ever need it in the future.
